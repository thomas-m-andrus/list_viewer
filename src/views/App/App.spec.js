import { shallowMount, mount} from '@vue/test-utils';
import VueRouter from 'vue-router';
import Vue from 'vue';
import AppComponent from './index.vue';


// Here are some Jasmine 2.0 tests, though you can
// use any test runner / assertion library combo you prefer
describe('AppComponent', () => {

})