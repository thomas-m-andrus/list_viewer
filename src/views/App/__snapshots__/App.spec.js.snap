// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent renders the component 1`] = `
<v-app>
  <v-toolbar app="">
    <v-toolbar-title class="headline text-uppercase"><span>Simple Vue List</span></v-toolbar-title>
    <v-spacer></v-spacer> <span class="mr-2">Latest Release</span>
  </v-toolbar>
  <v-content>
    <router-view></router-view>
  </v-content>
</v-app>
`;
